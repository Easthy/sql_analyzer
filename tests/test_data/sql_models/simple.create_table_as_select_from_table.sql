CREATE TABLE simple.create_table_as_select_from_table AS
SELECT id AS user_id,
       name
  FROM source.users;