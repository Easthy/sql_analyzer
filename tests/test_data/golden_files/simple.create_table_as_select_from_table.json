{
  "nodes": [
    { "id": "col:simple.create_table_as_select_from_table.name" },
    { "id": "col:simple.create_table_as_select_from_table.user_id" },
    { "id": "tbl:simple.create_table_as_select_from_table" },
    { "id": "tbl:source.users" }
  ],
  "links": [
    { "source": "tbl:simple.create_table_as_select_from_table", "target": "col:simple.create_table_as_select_from_table.name", "type": "contains_column" },
    { "source": "tbl:simple.create_table_as_select_from_table", "target": "col:simple.create_table_as_select_from_table.user_id", "type": "contains_column" },
    { "source": "tbl:simple.create_table_as_select_from_table", "target": "tbl:source.users", "type": "table_dependency" }
  ]
}